Dockerfile

```diff
@@ -2,6 +2,17 @@ FROM lsstsqre/centos:w_latest
 
 MAINTAINER Kenny Lo <kennylo@slac.stanford.edu>
 
+# Add user for testing in jenkins
+USER root
+ARG UNAME=jenkins
```

Comment:
I think `UNAME` is a misleading variable name as `uname` is a well known command.  I suggest changing this to `USER`.